src/Tools/GenerateProps/targetgroups.props

```diff
@@ -74,6 +74,9 @@
     </TargetGroups>
     <TargetGroups Include="netstandard">
       <NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
+      <!-- By default, netstandard libraries will be held to the UWP compatibility bar;
+           set this to false in the project to force otherwise. -->
+      <EnableWinRT>true</EnableWinRT>
```

Comment:
Yes it's a poor name, It comes from https://github.com/dotnet/corefx/blob/master/src/Tools/GenerateProps/targetgroups.props. I can rename it to eg `<UWPCompatible> `
As well as enabling`<EnablePinvokeUWPAnalyzer>` it also enables the `BlockReflectionAttribute`. If the distinction is not interesting I can eliminate `EnablePinvokeUWPAnalyzer`